### DISCUSSION TITLE
Markdown editor does not respect html to html render settings

### DISCUSSION THREAD

**Discussion by tionis** [requarks/wiki#3601] [Error / Bug Report]
**Bug-Description:**
As a workaround for #1547 and #1201 is switched on the html to html render setting "Treat relative links as root absolute". While the rendered output works, the live markdown preview does not respect this setting and simply doesn't show linked pictures and such.

**To Reproduce:**
1. Switch on "Treat relative links as root absolute" in the html to html render pipeline
2. Add a picture on the root of the wiki
3. Reference it as name.jpg (not /name.jpg as this would be absolute)

**Expected behavior:**
The Picture should show as normal (as it is shown when referencing it absolutely)

**Host Info:**
 - OS: Docker Container (Linux)
 - Wiki.js version: 2.4.107
 - Database engine: PostgreSQL 11.8

**Comment by tionis** [comment]
I've looked a bit more into it and it seems that relative links are simply broken in some ways in the preview and in other ways (as tracked in the mentioned issues) in the rendered html.
When i switch the html to html render option of, I can achieve my goal of relative links with the ./LINK_HERE notation, but it doesn't work on the live render of the markdown editor

**Comment by nwcm** [comment]
This is indeed still a bug in `2.5.300`. 

Relative images in markdown `./foo.png` will not render in the editor view.
